js/src/dom/eventHandler.js

```diff
@@ -197,21 +198,18 @@ const EventHandler = {
     }
 
     if (typeof handler === 'undefined') {
-      for (const elementEvent in events) {
-        if (!Object.prototype.hasOwnProperty.call(events, elementEvent)) {
+      const storeElementEvent = events[typeEvent]
+      for (const keyHandlers in storeElementEvent) {
+        if (!Object.prototype.hasOwnProperty.call(storeElementEvent, keyHandlers)) {
           continue
         }
 
-        const storeElementEvent = events[elementEvent]
-        for (const keyHandlers in storeElementEvent) {
-          if (!Object.prototype.hasOwnProperty.call(storeElementEvent, keyHandlers)) {
-            continue
-          }
-          // delete all the namespaced listeners
-          if (inNamespace && keyHandlers.indexOf(originalTypeEvent) > -1) {
-            const handlerFn = events[elementEvent][keyHandlers]
-            EventHandler.off(element, elementEvent, handlerFn.originalHandler)
-          }
+        const handlerKey = keyHandlers.replace(stripUidRegex, '')
```

Comment:
Why did you add a new regex here ?  `off` method already work

Reply:
`const` does not work in IE10

Reply:
We use babel.

Reply:
The `off` implementation was not complete and i think i've messed up some things. Now working on a cleaner and more jQuery-like solution.